environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
According to code in `build.js`
- If they have same `full`(name) or `family` then they are different versions of same thing
- `Object.keys()` is used for order so newer one should go after previous one